### DISCUSSION TITLE
[Feature] The mobile application does not support viewing photos and videos in 360 (panorama)

### DISCUSSION THREAD

**Discussion by FreeWind6** [immich-app/immich#13286] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The mobile application does not support viewing photos and videos in 360 (panorama)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6572
